security/pfSense-pkg-acme/files/usr/local/pkg/acme/acme_gui.inc

```diff
@@ -0,0 +1,139 @@
+<?php
+/*
+ * acme_gui.inc
+ * 
+ * part of pfSense (https://www.pfsense.org/)
+ * Copyright (c) 2016 PiBa-NL
+ * All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/* Include the autoloader for all the GUI display classes */
+require_once("classes/autoload.inc.php");
+
+$acme_icons = array(
+	'moveup' => array(
+		"faicon" => "fa-arrow-up",
+		"icon" => "icon_up.gif",
+		"iconsize" => 17),
+	'movedown' => array(
+		"faicon" => "fa-arrow-down",
+		"icon" => "icon_down.gif",
+		"iconsize" => 17),
+	'add' => array(
+		"faicon" => "fa-level-down",
```

Comment:
Single add buttons should use fa-plus, not fa-level-down. Only use fa-level-down when the page supports adding to the top or bottom of a list (For example, when there is also an fa-level-up add button).